### ISSUE TITLE
Prevent installing with mrt

### ISSUE THREAD

**Issue by aramk** [arunoda/laika#150]
I tend to develop meteor packages locally by creating symbolic from the `packages` directory. When running `mrt`, these are removed, but since I use Meteor's own package manager, there's no need to run `mrt` anyway. The issue is that Laika always runs Meteor through `mrt`, removing these symlinks. It would be nice to have a `no-mrt` option to avoid this and run `meteor` directly.

**Comment by apendua** [comment]
I think it depends on the existence of smart.json file in your project root
directory. Can you try removing it and see what happens? If it does not
help we can surely implement the solution your talking about.

W dniu sobota, 13 grudnia 2014 Aram Kocharyan notifications@github.com
napisał(a):

**Comment by aramk** [comment]
Hi @apendua. Removing `smart.json` works as a temporary solution as long as it's added back after running the tests. It still has the purpose of specifying the external, unpublished packages (most of them forks) used in the project, so removing them permanently isn't an option. I suspect replacing the invocation of `mrt` to `meteor` within laika if an option is provided would work well.

**Comment by apendua** [comment]
@aramk I agree. Can you implement this fix and submit a PR? I would also suggest looking into another testing solution which I am currently working on:
https://github.com/anticoders/gagarin
